packages/preset-mini/src/preflights.ts

```diff
@@ -8,7 +8,11 @@ export const preflights: Preflight[] = [
     getCSS(ctx: PreflightContext<Theme>) {
       if (ctx.theme.preflightBase) {
         const css = entriesToCss(Object.entries(ctx.theme.preflightBase))
-        return `*,::before,::after{${css}}::backdrop{${css}}`
+        let roots = toArray(ctx.theme.preflightRoot ?? [])
+        if (!roots.length)
+          roots = ['*,::before,::after', '::backdrop']
```

Comment:
```suggestion
        let roots = toArray(ctx.theme.preflightRoot ?? ['*,::before,::after', '::backdrop'])
```

Can we do this? I think it's also reasonable to pass an empty array to disable this preflight